### PR TITLE
Update viewMeeting command

### PR DESCRIPTION
**Pull request by Thing1Thing2**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ViewMeetingCommandParser.java [hunk 1]
```diff
@@ -17,12 +18,17 @@ public class ViewMeetingCommandParser implements Parser<ViewMeetingCommand> {
...
-            return new ViewMeetingCommand(index);
+            index = ParserUtil.parseIndex(argumentMultimap.getValue(PREFIX_INDEX).get());
         } catch (ParseException pe) {
-            throw new ParseException(
-                    String.format(MESSAGE_INVALID_COMMAND_FORMAT, ViewMeetingCommand.MESSAGE_USAGE), pe);
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    ViewMeetingCommand.MESSAGE_USAGE), pe);
         }
+
+        return new ViewMeetingCommand(index);
```
Comment (rexong) [inline src/main/java/seedu/address/logic/parser/ViewMeetingCommandParser.java:31]: Do check these two lines coverage. I checked on mine and the line coverage is 71%.

**Review by rexong — CHANGES_REQUESTED** [review]
The coverage not complete from my side. Do take a look again. 

ViewMeetingCommand `line 45-54` is also not covered. (Couldnt comment on these lines as they were not edited.)

File: src/main/java/seedu/address/logic/parser/ViewMeetingCommandParser.java (hunk 1 above)
Comment (Thing1Thing2) [reply src/main/java/seedu/address/logic/parser/ViewMeetingCommandParser.java:31]: Should be completed now, please review. Thank you 😄

**Review by rexong — APPROVED** [review]
LGTM

**Pull request merged**